nixos/modules/services/system/nscd.conf

```diff
@@ -4,25 +4,41 @@ paranoia                no
 debug-level             0
 
 enable-cache            passwd          yes
```

Comment:
do we want to disable caches where we set a zero ttl for both positive and negative? Seems less confusing to me :-)

Reply:
Oh I misread: See the original commit message:

```
Note that we can not just put in /etc/nscd.conf:
enable-cache passwd no

As this will actually cause glibc to _not_ forward the call to nscd
at all, and thus never reach the nss modules. Instead we set
the negative and positive cache ttls  to 0 seconds as a workaround.
This way, Glibc will always forward requests to nscd, but results
will never be cached.
```

Reply:
I added this as a comment to `nscd.conf` to clarify